iothub/device/samples/solutions/PnpDeviceSamples/Thermostat/ThermostatSample.cs

```diff
@@ -80,7 +80,7 @@ public async Task PerformOperationsAsync(CancellationToken cancellationToken)
             });
 
             _logger.LogDebug($"Set handler to receive \"targetTemperature\" updates.");
-            await _deviceClient.SetDesiredPropertyUpdateCallbackAsync(TargetTemperatureUpdateCallbackAsync, cancellationToken);
+            //await _deviceClient.SetDesiredPropertyUpdateCallbackAsync(TargetTemperatureUpdateCallbackAsync, cancellationToken);
```

Comment:
Plans for this?

Reply:
thanks for catching this; I forgot to uncomment this